src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey>true</UseECMAKey>
+    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
We are trying to preserve the assembly identity of this assembly which has shipped already in .NET and the identity includes the public key which was ECMA in this case.